C/plugins/south/modbus/plugin.cpp

```diff
@@ -23,8 +23,8 @@ using namespace std;
 /**
  * Default configuration
  */
-#define CONFIG	"{\"plugin\" : { \"description\" : \"Modbus TCP and RTU plugin\", " \
-			"\"type\" : \"string\", \"default\" : \"foglamp-modbus\" }, " \
+#define CONFIG	"{\"plugin\" : { \"description\" : \"Modbus TCP and RTU C plugin\", " \
+			"\"type\" : \"string\", \"default\" : \"modbus\" }, " \
```

Comment:
We can't just change the name here without chaining the name we build something as, it will break things

Reply:
I changed default plugin value to same as directory name for C plugins; because binaries name with "lib<directory_name>.so". 
Is there any standard follow for C-plugin default value? As we have for python plugins default value same as directory name and corresponding its py file name (to actually add the service)
I see except the `usb4704` c-south plugin, rest of the plugin starts with "foglamp-"